authorization/http_server_test.go

```diff
@@ -144,65 +144,57 @@ func TestService_handlePostAuthorization(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			t.Helper()
+		for _, useHashedTokens := range []bool{false, true} {
+			t.Run(fmt.Sprintf("%s/TokenHashing=%t", tt.name, useHashedTokens), func(t *testing.T) {
+				s := itesting.NewTestInmemStore(t)
 
-			s := itesting.NewTestInmemStore(t)
-			storage, err := NewStore(s)
-			if err != nil {
-				t.Fatal(err)
-			}
+				storage, err := NewStore(context.Background(), s, useHashedTokens)
+				require.NoError(t, err)
```

Comment:
Add some context to this error so we know what failed 
```go
require.NoError(t, err, "creating new store")
```

Reply:
This wouldn't add value since its repeating the line above.